cmd2/argparse_completer.py

```diff
@@ -695,7 +695,7 @@ def basic_complete(text: str, line: str, begidx: int, endidx: int, match_against
 
 
 # noinspection PyCompatibility,PyShadowingBuiltins,PyShadowingBuiltins
-class ACHelpFormatter(argparse.HelpFormatter):
+class ACHelpFormatter(argparse.RawTextHelpFormatter):
```

Comment:
This looks like a pretty straightforward change to me and low risk since it would only affect end users currently using ``cmd2.argparse_completer.ACHelpFormatter`` which was added relatively recently.